cli/cmd/root.go

```diff
@@ -430,6 +430,25 @@ func printIfError(cmd *cobra.Command, err error) {
 		fmt.Fprintln(os.Stderr, fmt.Sprintf("METHOD: %s", err.Method))
 		fmt.Fprintln(os.Stderr, fmt.Sprintf("ENDPOINT: %s", err.Endpoint))
 		fmt.Fprintln(os.Stderr, err.Message) // note that this can have multiple lines
+	case ClusterTimeoutError:
+		fmt.Fprintf(os.Stderr, "ERROR: Wait timeout exceeded for cluster %s\n", err.Cluster.ID)
```

Comment:
nit: Just because other errors in the CLI use `Error:` instead of `ERROR:` perhaps keep continuity here.
```suggestion
		fmt.Fprintf(os.Stderr, "Error: Wait timeout exceeded for cluster %s\n", err.Cluster.ID)
```

I have no idea why this was previously upper case, for example:

```
r vm rm foobar
Error: VM with name or ID 'foobar' not found
```

Produces `Error: ...` text 🤷